opus/application/static_media/js/opus.js

```diff
@@ -238,10 +278,11 @@ var opus = {
          * Given the result of the search parameter normalization, execute the search
          * that will eventually return the result count.
          */
-
         // If there are more normalized data requests in the queue, don't trigger
         // spurious result counts that we won't use anyway
         if (normalizedData.reqno < opus.lastAllNormalizeRequestNo) {
+            opus.normalizeInputForAllFieldsInProgress[opus.allSlug] = false;
+            o_widgets.disableButtonsInAWidget(false);
```

Comment:
Should this be called disableButtonsInWidgets since it applies to all widgets, not a single widget?